schema/organization/Person.schema.yaml

```diff
@@ -2,27 +2,28 @@ title: Person
 '@id': schema:Person
 $extends: ../Thing.schema.yaml
 role: secondary
-status: unstable
+status: stable
 description: A person (alive, dead, undead, or fictional). https://schema.org/Person.
 parser: person
 properties:
+  address:
+    '@id': schema:address
+    description: Postal address for the person.
+    type: string
   affiliations:
     '@id': schema:affiliation
+    description: Organizations that the person is affiliated with.
     type: array
     items:
       $ref: Organization.schema.yaml
-  brand:
-    '@id': schema:brand
-    type: array
-    items:
-      $ref: Brand.schema.yaml
   contactPoint:
     '@id': schema:contactPoint
     type: object
     items:
       $ref: ContactPoint.schema.yaml
```

Comment:
I don't think we need `contactPoint` here. We now have `address` and `telephoneNumbers` which I think is more intuitive.

Reply:
Agreed.